config/checker_labels.json

```diff
@@ -0,0 +1,2682 @@
+{
+  "constraints": {
+    "profile": {
+      "choice": {
+        "default": "High-quality standard checks with a low false positive rate.",
+        "sensitive": "Default checks + more comprehensive checks with a low false positive rate.",
+        "security": "Checkers that warn for potentially vulnerable code. Contains all SEI Cert C/C++ rules.",
+        "portability": "Checks that aim to detect code issues emerging from platform differences (eg. between 32-bit and 64-bit architectures).",
+        "extreme": "Sensitive checks + more comprehensive checks with a manageable false positive rate."
+      }
+    },
+    "guideline": {
+      "choice": {
+        "sei-cert": "https://wiki.sei.cmu.edu/confluence/display/seccode"
+      }
+    },
+    "sei-cert": {
+
+    },
+    "severity": {
+      "choice": {
+        "STYLE": "A true positive indicates that the source code is against a specific coding guideline or could improve readability.",
+        "LOW": "A true positive indicates that the source code is hard to read/understand or could be easily optimized.",
+        "MEDIUM": "A true positive indicates that the source code that may not cause a run-time error (yet), but against intuition and hence prone to error.",
+        "HIGH": "A true positive indicates that the source code will cause a run-time error.",
+        "CRITICAL": "Currently used for indicating compilation errors.",
+        "UNSPECIFIED": "Checker severity is not specified for a checker."
+      },
+      "unique": true,
```

Comment:
Because the severity of a checker must be unique. If this constraint is violated, then CodeChecker throws an error in the very beginning. By the way you can use "required" constraint too. That would be useful if you want to force providing severities for each checker.